airbyte-integrations/connectors/source-faker/source_faker/spec.json

```diff
@@ -35,8 +35,16 @@
         "description": "How many fake records will be in each page (stream slice), before a state message is emitted?",
         "type": "integer",
         "minimum": 1,
-        "default": 100,
+        "default": 1000,
         "order": 3
+      },
+      "threads": {
+        "title": "Worker Threads",
+        "description": "How many parallel worker threads should be used to generate records?",
```

Comment:
is the user really in a position to tell us what this number should be? can't we make a best effort guess internally? even as an engineer I might be confused at what this number should be.